docs/api.md

```diff
@@ -44,375 +44,378 @@
-  * [httpResponse.buffer()](#httpresponsebuffer)
-  * [httpResponse.frame()](#httpresponseframe)
-  * [httpResponse.fromCache()](#httpresponsefromcache)
-  * [httpResponse.fromServiceWorker()](#httpresponsefromserviceworker)
-  * [httpResponse.headers()](#httpresponseheaders)
-  * [httpResponse.json()](#httpresponsejson)
-  * [httpResponse.ok()](#httpresponseok)
-  * [httpResponse.remoteAddress()](#httpresponseremoteaddress)
-  * [httpResponse.request()](#httpresponserequest)
-  * [httpResponse.securityDetails()](#httpresponsesecuritydetails)
-  * [httpResponse.status()](#httpresponsestatus)
-  * [httpResponse.statusText()](#httpresponsestatustext)
-  * [httpResponse.text()](#httpresponsetext)
-  * [httpResponse.url()](#httpresponseurl)
-- [class: SecurityDetails](#class-securitydetails)
-  * [securityDetails.issuer()](#securitydetailsissuer)
-  * [securityDetails.protocol()](#securitydetailsprotocol)
-  * [securityDetails.subjectAlternativeNames()](#securitydetailssubjectalternativenames)
-  * [securityDetails.subjectName()](#securitydetailssubjectname)
-  * [securityDetails.validFrom()](#securitydetailsvalidfrom)
-  * [securityDetails.validTo()](#securitydetailsvalidto)
-- [class: Target](#class-target)
-  * [target.browser()](#targetbrowser)
-  * [target.browserContext()](#targetbrowsercontext)
-  * [target.createCDPSession()](#targetcreatecdpsession)
-  * [target.opener()](#targetopener)
-  * [target.page()](#targetpage)
-  * [target.type()](#targettype)
-  * [target.url()](#targeturl)
-  * [target.worker()](#targetworker)
-- [class: CDPSession](#class-cdpsession)
-  * [cdpSession.connection()](#cdpsessionconnection)
-  * [cdpSession.detach()](#cdpsessiondetach)
-  * [cdpSession.id()](#cdpsessionid)
-  * [cdpSession.send(method[, ...paramArgs])](#cdpsessionsendmethod-paramargs)
-- [class: Coverage](#class-coverage)
-  * [coverage.startCSSCoverage([options])](#coveragestartcsscoverageoptions)
-  * [coverage.startJSCoverage([options])](#coveragestartjscoverageoptions)
-  * [coverage.stopCSSCoverage()](#coveragestopcsscoverage)
-  * [coverage.stopJSCoverage()](#coveragestopjscoverage)
-- [class: TimeoutError](#class-timeouterror)
-- [class: EventEmitter](#class-eventemitter)
-  * [eventEmitter.addListener(event, handler)](#eventemitteraddlistenerevent-handler)
-  * [eventEmitter.emit(event, [eventData])](#eventemitteremitevent-eventdata)
-  * [eventEmitter.listenerCount(event)](#eventemitterlistenercountevent)
-  * [eventEmitter.off(event, handler)](#eventemitteroffevent-handler)
-  * [eventEmitter.on(event, handler)](#eventemitteronevent-handler)
-  * [eventEmitter.once(event, handler)](#eventemitteronceevent-handler)
-  * [eventEmitter.removeAllListeners([event])](#eventemitterremovealllistenersevent)
-  * [eventEmitter.removeListener(event, handler)](#eventemitterremovelistenerevent-handler)
-- [interface: CustomQueryHandler](#interface-customqueryhandler)
+- [Puppeteer API Tip-Of-Tree](#puppeteer-api-tip-of-tree)
```

Comment:
the diff here looks wrong. What was actually changed here?

Reply:
oh, sorry I think my editor messed up the file, will fix